.github/workflows/post-merge-validation.yml

```
@@ -0,0 +1,145 @@
+        with:
+          python-version: '3.10'
+      
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r config/requirements.txt
+          pip install -r config/requirements-dev.txt
+      
+      - name: Create validation report
+        id: validation
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:7: Double quote to prevent globbing and word splitting

(shellcheck)